packages/SwingSet/package.json

```diff
@@ -24,8 +24,7 @@
     "@types/better-sqlite3": "^7.5.0",
     "@types/microtime": "^2.1.0",
     "@types/tmp": "^0.2.0",
-    "better-sqlite3": "^8.2.0",
-    "yargs-parser": "^21.1.1"
+    "better-sqlite3": "^8.2.0"
```

Comment:
Why is `"better-sqlite3": "^8.2.0"` in both `devDependencies` and `dependencies`?

Reply:
Thanks for catching. It was a devDependency for tests. Then `misc-tools` exported something that depends on it, so it needed to go in `dependencies`. 

I could imagine that there's a better place for those tools, but it wouldn't remove the dependency because it would still be included transitively by `@agoric/swingset-store`. So I'll just remove the redundant one in devDependencies.